### DISCUSSION TITLE
endpointing for German roadmap?

### DISCUSSION THREAD

**Discussion by 3adel** [deepgram/community#146] [General help] [answered]
Could you share info on your plans to make endpointing reliable for German. My streaming use case (using a mic on my PC or Phone from the browser) relies heavily on it working reliably, similar to the English counterpart. Thanks.

**Comment by shirgoldbird** [comment]
Hi @3adel! Could you share some more details on what you mean by making endpointing reliable for German? Our [Endpointing parameter](https://developers.deepgram.com/documentation/features/endpointing/) is designed to return transcripts as soon as possible after someone finishes speaking. However, background noise may interfere with it returning results promptly. If you tell me some more details about your use case and the issues you're encountering, I may be able to make a better suggestion.

**Reply by 3adel** [reply]
Thanks @shirgoldbird , it's the same exact issue highlighted here https://github.com/orgs/deepgram/discussions/138#discussion-5159263

I experimented with different audio streaming sources: A Macbook Pro internal mic as well as the iPhone mic. It's worth mentioning that this issue does not happen in English, but it does in German. Could you advise if the German model is still not as good as the English counterpart when it comes to detecting enpointing? Here's the URL for the endpoint I am using: `wss://api.deepgram.com/v1/listen?language=de&punctuate=true&endpointing=500`

**Reply by shirgoldbird** [reply, marked as answer]
@3adel It seems like our German model has a recurring issue where it detects words in silence. This is due to the data the model was trained on and can't be fixed on your end by configuring Deepgram's parameters.

 I've passed along your feedback to the research team so we can investigate improvements. Unfortunately, I don't have a timeline for you at the moment, but we usually announce language model updates on our [changelog](https://deepgram.com/changelog/) and [social media ](https://twitter.com/deepgramAI).

Please let me know if you have any other questions!
